### PR TITLE
Added recipe for ESPEI

### PR DESCRIPTION
**Pull request by bocklund**

### CODE REVIEW

File: recipes/espei/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (synapticarbors) [inline recipes/espei/meta.yaml]: Please remove all comments for final recipe.

File: recipes/espei/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    ESPEI, or Extensible Self-optimizing Phase Equilibria Infrastructure, 
+    is a tool for automated thermodynamic database development within the 
+    CALPHAD method.
+  doc_url: http://espei.org
+  dev_url: https://github.com/PhasesResearchLab/espei
+
+extra:
+  recipe-maintainers:
+    - bocklund
+    - richardotis
```
Comment (synapticarbors) [inline recipes/espei/meta.yaml]: @richardotis -- are you ok being added as a co-maintainer on the conda-forge package for `espei`?

File: recipes/espei/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```
Comment (synapticarbors) [inline recipes/espei/meta.yaml]: It looks like this package has a python entry_point:

https://github.com/PhasesResearchLab/ESPEI/blob/master/setup.py#L62-L63

Please add it to the `build` section:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#python-entry-points

File: recipes/espei/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    - dask >=0.15
+    - distributed
+    - tinydb >=3
+    - scikit-learn
+    - emcee
+    - pycalphad >=0.5
+    - cerberus
+    - mpi4py
+
+test:
```
Comment (synapticarbors) [inline recipes/espei/meta.yaml]: Please add a test for the command line tool tied to the `entry_point`:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#test-commands

File: recipes/espei/meta.yaml (hunk 2 above)
Comment (richardotis) [reply recipes/espei/meta.yaml]: I agree to be a co-maintainer of `espei` on conda-forge.

**Comment by bocklund** [comment]
@synapticarbors this is ready for another look

**Pull request merged**